tests/test-cases/Group1-Docker-Commands/1-10-Docker-PS.robot

```diff
@@ -71,7 +71,7 @@ Docker ps only running containers
     ${output}=  Split To Lines  ${output}
     ${len}=  Get Length  ${output}
     Create several containers
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps
+    ${rc}  ${output}=  Wait Until Keyword Succeeds  5x  5 seconds  Run And Return Rc And Output  docker %{VCH-PARAMS} ps
```

Comment:
but you aren't looking for anything? docker ps should always work, just the power state of some containers might be inconsistent for a while right?